src/alfanous-django/wui/templates/wui/result/header.html

```diff
@@ -1,5 +1,6 @@
 {% load query %}
 
+<h1>
```

Comment:
is that the page header or aya header?

Reply:
yes only one h1 for the page , an i think h2 for query section, keywords, suggestions, aya results ... and h3 for each aya result